src/utils/utils.js

```diff
@@ -65,3 +65,5 @@ export const getResource = (
 
   return res;
 };
+
+export const delay = ms => new Promise(resolve => setTimeout(resolve, ms));
```

Comment:
this might be not neccessary as we already have https://github.com/kubevirt/web-ui-components/blob/master/src/tests/enzyme.js#L3 or if it does not work for you than please move delay to `src/tests` dir

Reply:
removed